docs/notes.md

```diff
@@ -603,4 +603,254 @@ sends a pair \\((L\_j,R\_j)\\) of points at each step \\(j = k\dots1\\). An
+
+\\[
+\begin{aligned}
+  t(x) &= \sum_{j=0}^{m-1} t\_{(j)}(x)\\\\
+  {\tilde{t}}(x) &= \sum_{j=0}^{m-1}{\tilde{t}}\_{(j)}(x)\\\\
+  T_1 &= \sum_{j=0}^{m-1} T_{(j)1}\\\\
+  T_2 &= \sum_{j=0}^{m-1} T_{(j)2}\\\\
+  \delta(y,z) &= \sum_{j=0}^{m-1} \delta\_{(j)}(y,z)\\\\
+  &= (z - z^{2}) \cdot \sum_{j=0}^{m-1} {\langle {\mathbf{1}}, {\mathbf{y}}^{n}\_{(j)} \rangle} - z^{3} \sum_{j=0}^{m-1} z\_{(j)} \cdot {\langle {\mathbf{1}}, {\mathbf{2}}^{n \cdot m} \rangle}\\\\
+\end{aligned}
+\\]
+
+Since we know that \\(\mathbf{y}^n\_{(j)} = \mathbf{y}^{n \cdot m}\_{[j \cdot n : (j+1) \cdot n - 1]} \\), we can simplify \\(\delta(y, z)\\):
+
+\\[
+\begin{aligned}
+  \delta(y, z) &= (z - z^{2}) \cdot (
+    {\langle {\mathbf{1}}, \mathbf{y}^{n \cdot m}\_{[0 : n - 1]} \rangle + 
+    \langle {\mathbf{1}}, \mathbf{y}^{n \cdot m}\_{[n : 2 \cdot n - 1]} \rangle + 
+    \dots +
+    \langle {\mathbf{1}}, \mathbf{y}^{n \cdot m}\_{[(m-1) \cdot n : m \cdot n - 1]} \rangle}) -
+  z^{3} \sum_{j=0}^{m-1} z\_{(j)} \cdot {\langle {\mathbf{1}}, {\mathbf{2}}^{n \cdot m} \rangle} \\\\
+  &= (z - z^{2}) \cdot {\langle {\mathbf{1}}, \mathbf{y}^{n \cdot m} \rangle} - z^{3} \sum_{j=0}^{m-1} z\_{(j)} \cdot {\langle {\mathbf{1}}, {\mathbf{2}}^{n \cdot m} \rangle} \\\\
+\end{aligned}
+\\]
+
+
+Now instead of having to do `m` individual checks to prove that \\(t\_{(j)0}\\) for all parties `j` are correct, we can do the verification with one check using the combined values:
+
+\\[
+\begin{aligned}
+  t(x) B + {\tilde{t}}(x) {\widetilde{B}} \stackrel{?}{=} z^2 \sum_{j=0}^{m-1} z\_{(j)} V_{(j)} + \delta(y,z) B + x T\_{1} + x^{2} T\_{2},\\\\
+  \delta(y,z) = (z - z^{2}) \cdot {\langle {\mathbf{1}}, {\mathbf{y}}^{n \cdot m} \rangle} - z^{3} \sum_{j=0}^{m-1} z\_{(j)} \cdot {\langle {\mathbf{1}}, {\mathbf{2}}^{n \cdot m} \rangle}\\\\
+\end{aligned}
+\\]
+
+Since we know that \\(z\_{(j)} = z^j\\), we can rewrite the equation as follows:
+
+\\[
+\begin{aligned}
+  t(x) B + {\tilde{t}}(x) {\widetilde{B}} \stackrel{?}{=} \sum_{j=0}^{m-1} z^{j+2} V_{(j)} + \delta(y,z) B + x T\_{1} + x^{2} T\_{2},\\\\
+  \delta(y,z) = (z - z^{2}) \cdot {\langle {\mathbf{1}}, {\mathbf{y}}^{n \cdot m} \rangle} - \sum_{j=0}^{m-1} z^{j+3} \cdot {\langle {\mathbf{1}}, {\mathbf{2}}^{n \cdot m} \rangle}\\\\
+\end{aligned}
+\\]
+
+Proving that \\({\mathbf{l}}(x)\\), \\({\mathbf{r}}(x)\\) are correct
+---------------------------------------------------------------------
+
+Proving that \\({\mathbf{l}}\_{(j)}(x)\\), \\({\mathbf{r}}\_{(j)}(x)\\) are correct requires first creating commitments to the variables, and then proving a relation over the commitments. For an explanation of how the commitments are created and how the relation is derived, see the [proving that \\({\mathbf{l}}(x)\\), \\({\mathbf{r}}(x)\\) are correct](index.html#proving-that-mathbflx-mathbfrx-are-correct) step of the single-value range proof. The relation over the commitments to prove is:
```

Comment:
"The relation over the commitments to prove" is a little awkward, maybe "The statement each party wants to prove is" (since I think (from the subscripts) that this is a per-party statement that hasn't been combined yet?)